homeassistant/helpers/template.py

```diff
@@ -1350,9 +1350,9 @@ def from_json(value):
     return json.loads(value)
 
 
-def to_json(value):
+def to_json(value, ensureascii=True):
```

Comment:
Separate words with an underscore.

```suggestion
def to_json(value, ensure_ascii=True):
```